thesis-template/template.yml

```diff
@@ -2,7 +2,7 @@ myst: v1
 kind: typst
 title: Thesis Template
 description: Typst-MyST Thesis Template
-template: typst-thesis.typ
+template: template.typ
```

Comment:
This was the main reason things were blank, the typst-thesis.typ builds a blank PDF always.